browser/frontend/lambdas/chalice/browser-frontend/Makefile

```diff
@@ -0,0 +1,39 @@
+SHELL=/bin/bash -o pipefail
+
+# This is set for build reproducibility
+export ACCOUNT_ID=$(shell aws sts get-caller-identity --query Account --output text)
+export APP_NAME?=browser-frontend
+ifndef DEPLOYMENT_STAGE
+$(error Please set DEPLOYMENT_STAGE in environment before running make commands)
+endif
+export EXPORT_ENV_VARS_TO_LAMBDA=APP_NAME DEPLOYMENT_STAGE
+
+.PHONY: clean
+clean:
+	rm -rf dist .chalice/deployments chalicelib .chalice/config.json .chalice/policy-*
+
+.PHONY: destroy
+destroy:
+	chalice delete --stage $(DEPLOYMENT_STAGE
+
+.PHONY: package
+package: build-chalice-config
+	mkdir -p chalicelib/code
+	cp -R ../../code chalicelib
+
+.PHONY: deploy
+deploy: package
+	chalice deploy --stage $(DEPLOYMENT_STAGE) --api-gateway-stage $(DEPLOYMENT_STAGE)
```

Comment:
Is `--api-gateway-stage` still required?

Reply:
Nope, I'll remove